README.md

```diff
@@ -26,16 +26,30 @@ pnpm preview
 
 ## Catalog
 
-In [catalog.yml](catalog.yml) you can find the list of books and chapters that can be combined.
+In [books.yml](books.yml) you can find the list of books that can be combined.
 
-In [catalog.json](catalog.json) you can find the same books, but with their table of contents fetched.
+In [chapters.json](chapters.json) you can find the same books, but with their table of contents fetched.
 You should not edit this file manually, as it is generated by the `harvest.py` script.
 
-The `catalog.json` file is generated by running the following command:
+The `chapters.json` file is generated by running the following command:
 
 ```bash
 # Install the dependencies, if you haven't already
-pip install -e .
-./harvest.py
-# The catalog.json file will be generated
-```
\ No newline at end of file
+pnpm install
+pnpm harvest
+# Reads books.yml and generates chapters.json
+```
+
+## Remote catalog
+
+If you want to show other books the then ones specified in the `./books.yml` file, you can point the app to a remote
```

Comment:
Also a nice additional feature! Shame that it doesn't work on gitlab due to the invalid cors headers...